JavaFXUserInterface/src/main/java/Controllers/AddSystemController.java

```diff
@@ -17,6 +19,11 @@
 
 public class AddSystemController implements Initializable {
 
+    private static final String SAVE_DIALOG = "Save Dialog";
+    private static final String SAVE_HEADER = "Asset has been saved to the database.";
+    private static final String ERROR_DIALOG = "Error Dialog";
+    private static final String ERROR_HEADER = "Please enter values for all text fields.";
+    private static final String SYSTEMS = "/Systems";
```

Comment:
if we want to do this, then it should be moved to the TextConstant file or another constant file

Reply:
I wanted to leave it for the code convention enhancement issue. But if I am gonna touch on it, I can pretty much do it at this point, I'll add it.

Reply:
ok, just leave if for that issue then